test/controllers/admin/comfy/external_api_clients_controller_test.rb

```diff
@@ -590,4 +590,186 @@ class Comfy::Admin::ExternalApiClientsControllerTest < ActionDispatch::Integrati
     assert_match expected_message, vacuum_job_plugin.reload.error_message
     assert_response :redirect
   end
+
+  # SyncAttributeToApiNamespace Plugin
+  test "#sync_attribute_to_api_namespace_plugin: adds provided new attribute to api namespace and backfills the default value to its api-resources successfully" do
```

Comment:
@alis-khadka the new location for plugin tests is `test/plugins` for example: https://github.com/restarone/violet_rails/blob/master/test/plugins/mailchimp_plugin_test.rb